site/content/guides/fips.md

```diff
@@ -47,8 +47,9 @@ The Contour [Dockerfile][8] uses a multistage build that performs compilation in
 In order to minimize the `projectcontour/contour` image footprint, the final output image only consists of a single layer, containing a lone file: the statically compiled `contour` binary.
 The standard Contour build uses the upstream `golang` image as a build base, however we will have to swap that out to build Contour with BoringCrypto.
 
-We can use the Google-provided Go implementation that has patches on top of standard Go to enable integrating BoringCrypto.
-This is available to us in the [`goboring/golang`][9] container image we can use as a build base.
+Starting with Go 1.19, you can simply add [`GOEXPERIMENT=boringcrypto`][18] to enable integrating BoringCrypto for standard Go. For the Go version under 1.19, we can use the Google-provided Go implementation that has patches on top of standard Go to enable integrating BoringCrypto.
```

Comment:
It'd be great if you could add an example of the command needed when using this Go 1.19 method. One option for organization would be to split the `Building Contour` section into multiple subsections, i.e. one for Go 1.19+ and one for older Go versions.

Reply:
note the additional flags and enabling CGO from previous versions of Go don't seem to be necessary, so having two separate sections to make that clear would be nice

Reply:
Thanks for the suggestion and supplement. 👍  I will add them to the doc later.
But with CGO enabled, I got this output running `goversion` for inspection:

> ./contour go1.19.2 X:boringcrypto (boring crypto)

while with CGO disabled:

>./contour go1.19.2 X:boringcrypto (standard crypto)

So it seems that CGO enabled is still necessary for `boringcrypto`. And the same as the additional flags for CGO enabled: 

> BUILD_EXTRA_GO_LDFLAGS="-linkmode=external -extldflags=-static"

Reply:
ah good call, I was passing the `-v` flag not `-crypto`, running:

```
$ make container BUILD_GOEXPERIMENT=boringcrypto BUILD_CGO_ENABLED=1 BUILD_EXTRA_GO_LDFLAGS="-linkmode=external -extldflags=-static"
```

you should get the proper output:

```
$ goversion -crypto ./contour
./contour go1.19.2 X:boringcrypto (boring crypto)
```

Reply:
no issues so far, so maybe can be ignored, but something to keep an eye on if there are any other relevant warnings in the future